### PR TITLE
Tp sendgrid

### PR DESCRIPTION
**Pull request by capricornication**
(_)_)

### CODE REVIEW

File: app/mailers/user_notifier_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class UserNotifierMailer < ApplicationMailer
+
+
+	default :from => 'company@sparkstart.org'
+
+	# send a signup email to the user, pass in the user object that   contains the user's email address
+	def send_signup_email(user)
+	  @user = user
+	  mail( :to => @user.email,
+	  :subject => 'Thanks for signing up for Spark Start!' )
```
Comment (stefanvlahov) [inline app/mailers/user_notifier_mailer.rb:10]: Can the :subject symbol be easily changed going forward? Without getting into the back-end?

File: spec/mailers/user_notifier_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe UserNotifierMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (stefanvlahov) [inline spec/mailers/user_notifier_spec.rb:5]: Noice!!!

**Comment by stefanvlahov** [comment]
Nice job! Squish (probably) and do the other thing ;)

**Comment by JenWestfallen** [comment]
Way to go!

**Pull request closed without merging**